openapiart/bundler.py

```diff
@@ -892,6 +965,7 @@ def _generate_value_schema(
                     "length": {
                         "description": "Number of bits to track for metrics starting from configured offset of corresponding header field",
                         "type": "integer",
+                        "format": "uint32",
```

Comment:
why is this hardcoded ? shouldn't it depend on bit length ?

Reply:
done